app/src/androidTest/java/io/mattcarroll/androidtesting/login/EspressoSignInTest.java

```diff
@@ -0,0 +1,103 @@
+package io.mattcarroll.androidtesting.login;
+
+
+import android.support.test.rule.ActivityTestRule;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import io.mattcarroll.androidtesting.BaseTest;
+import io.mattcarroll.androidtesting.R;
+import io.mattcarroll.androidtesting.SplashActivity;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.scrollTo;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public class EspressoSignInTest extends BaseTest{
+
+    @Rule
+    public final ActivityTestRule<SplashActivity> activityRule=
+            new ActivityTestRule<>(SplashActivity.class, false, true);
+
+    @Test
+    public void userSignInPersonalInfoRequiredFieldsAreRequired(){
```

Comment:
Unfortunately, your logic breaks this test! You are doing Sign In in  Before annotation , and this test requires no sign in ( this is negative sign in test)